easybuild/easyblocks/p/pytorch.py

```diff
@@ -89,9 +89,9 @@ def is_version_ok(version_range):
             """Return True if the PyTorch version to be installed matches the version_range"""
             min_version, max_version = version_range.split(':')
             result = True
-            if min_version and pytorch_version < LooseVersion(min_version):
+            if min_version and pytorch_version < min_version:
                 result = False
-            if max_version and pytorch_version >= LooseVersion(max_version):
+            if max_version and pytorch_version >= max_version:
```

Comment:
@Flamefire Did you check if automatically "casting" a string value to a `LooseVersion` object is also done in Python 2.7 when versions are being compared?

Reply:
Yes:
```
$ python2 
Python 2.7.18 (default, Jul  1 2022, 12:27:04) 
[GCC 9.4.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from distutils.version import LooseVersion
>>> LooseVersion('1.0') == '1.0'
True
>>> LooseVersion('1.0') != '1.0'
False
```